index.js

```diff
@@ -27,6 +27,7 @@ var Session = require('./session/session')
   , Cookie = require('./session/cookie')
   , Store = require('./session/store')
 
+var parseDomain = require('parse-domain');
```

Comment:
If you can, please move this up with the other requires (probably right before the `parseUrl` one).